### PR TITLE
Learning objectives styling

### PR DESCRIPTION
**Pull request by helenemccarron**

### CODE REVIEW

File: styles/designs/carnival/parts/_learning-components.scss [hunk 1]
```diff
@@ -28,25 +32,14 @@ $LearningListComponent: (
...
+    padding: enum('ValueSet:::OPTIONAL'),
   )
 );
 
-// HTML is annoying and you can't just simply change the bullet color
-// This is doing that manually
 $LearningListDotComponent: (
-  _name: 'ListDot',
-  _subselector: '::before',
+  _name: 'listDot',
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_learning-components.scss]: The linter will complain about this. Why was it changed back?

File: styles/designs/carnival/parts/_learning-components.scss [hunk 2]
```diff
@@ -28,25 +32,14 @@ $LearningListComponent: (
...
+  _subselector: '::marker',
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_learning-components.scss]: Good catch!

File: styles/output/anatomy-pdf.css [hunk 3]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 @charset "UTF-8";
+=======
+@import url("https://fonts.googleapis.com/css?family=IBM+Plex+Sans&display=swap");
+>>>>>>> fix LOs
```
Comment (tomjw64) [inline styles/output/anatomy-pdf.css]: Looks like a git merge went wrong! Was the style rebuilt?

File: styles/templates/template1/_colors.scss [hunk 4]
```diff
@@ -28,7 +28,7 @@
     G5:    #479177,
     G6:    #78B042,
     Grey1: #EEEAE0,
-    bodyFontColor: #333333,
+    bodyFontColor: #000000,
```
Comment (tomjw64) [inline styles/templates/template1/_colors.scss]: I can't seem to find the spec for what this value should be. Do you have a link?

Comment (helenemccarron) [reply styles/templates/template1/_colors.scss]: From https://drive.google.com/drive/folders/1gXs4VUFDNWzvxFvU7cDEw9qtllzJjVRX the initial value #333 was slapped on by me early on and totally arbitrary.

File: styles/designs/carnival/parts/_learning-components.scss (hunk 1 above)
Comment (helenemccarron) [reply styles/designs/carnival/parts/_learning-components.scss]: oops, not sure, probably from rebasing a few times, fixing now.

File: styles/output/anatomy-pdf.css (hunk 3 above)
Comment (helenemccarron) [reply styles/output/anatomy-pdf.css]: yep, but it looks like my merge wasn't as clean as it should, fixing now.

**Pull request merged**